bottomsheet-commons/src/main/java/com/flipboard/bottomsheet/commons/ImagePickerSheetView.java

```diff
@@ -501,6 +509,18 @@ public Builder setPickerDrawable(@DrawableRes int resId) {
         }
 
         /**
+         * Sets a layout for the image tile. Default is to use plain image view included in the
+         * library.
```

Comment:
You should add that the root of the layout must be an `ImageView`, or the sheet will crash at runtime.

Reply:
@markrietveld Resolved and also added a type check.